### ISSUE TITLE
Sequence mask in Tacotron2

### ISSUE THREAD

**Issue by CrossEntropy** [TensorSpeech/TensorFlowTTS#439]
Hi @dathudeptrai Thank you for your open source contribution! Now, I have some questions about making input sequence mask in `TFTacotron2 call`(https://github.com/TensorSpeech/TensorFlowTTS/blob/master/tensorflow_tts/models/tacotron2.py#L810) . 

  During training, we padd the input character sequence, and at the same time, `input_lengths` stores the true length of the character sequence. `maxlen=tf.reduce_max(input_lengths)` Will this cause the length of the mask to be less than the step number of the bidirectional lstm? Thank you for your answer!

**Comment by dathudeptrai** [comment]
pls see official doc here (https://www.tensorflow.org/api_docs/python/tf/keras/layers/LSTM?version=nightly):

```
mask: Binary tensor of shape [batch, timesteps] indicating whether a given timestep should be masked (optional, defaults to None). An individual True entry indicates that the corresponding timestep should be utilized, while a False entry indicates that the corresponding timestep should be ignored.
```

**Comment by CrossEntropy** [comment]
I have learned it, thanks for your reply!
